[2025-09-08T17:02:00.515+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:02:00.525+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskC scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-09-08T17:02:00.530+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskC scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-09-08T17:02:00.530+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:02:00.543+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): taskC> on 2025-09-04 00:00:00+00:00
[2025-09-08T17:02:00.546+0000] {standard_task_runner.py:72} INFO - Started process 16423 to run task
[2025-09-08T17:02:00.549+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskC', 'scheduled__2025-09-04T00:00:00+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpncobc7gj']
[2025-09-08T17:02:00.550+0000] {standard_task_runner.py:105} INFO - Job 122: Subtask taskC
[2025-09-08T17:02:00.576+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskC scheduled__2025-09-04T00:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:02:00.623+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskC' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-04T00:00:00+00:00'
[2025-09-08T17:02:00.624+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:02:00.634+0000] {logging_mixin.py:190} INFO - TASK C executed!
[2025-09-08T17:02:00.634+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-08T17:02:00.639+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:02:00.639+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskC, run_id=scheduled__2025-09-04T00:00:00+00:00, execution_date=20250904T000000, start_date=20250908T170200, end_date=20250908T170200
[2025-09-08T17:02:00.681+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:02:00.694+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:02:00.698+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
